packages/cxl-lumo-styles/scss/typography.scss

```diff
@@ -48,6 +48,7 @@ h3,
 h4,
 h5,
 h6 {
+  font-family: Raleway, sans-serif;
```

Comment:
I suggest to make `--cxl-lumo-font-ternary` variable so we have all font faces listed at glance in the top of the stylesheet.

Reply:
Updated